src/shared/components/results/ResultsData.tsx

```diff
@@ -108,6 +119,35 @@ const ResultsData: FC<{
     prevViewMode.current !== viewMode ||
     prevNamespace.current !== namespace;
 
+  const finalColumns = useMemo(() => {
```

Comment:
This use case is very similar to the ID Mapping "extra" column, so I wonder if it would be possible to put that logic in the same place in `useColumns`? see the use of the `displayIdMappingColumns` flag here: https://github.com/ebi-uniprot/uniprot-website/pull/417/files?file-filters%5B%5D=.tsx&hide-deleted-files=true#diff-2574fa071caa07abbdde4d341fd363f480b8b0d44fbbe032721f33b5c18d57f4R146

Reply:
Done